internal/wrappers/headers/kuzzlesdk.h

```diff
@@ -396,6 +396,13 @@ typedef struct {
     char *stack;
 } int_result;
 
+typedef struct date_result {
+    long long result;
```

Comment:
Since we have a dedicated result structure for dates, then I would use one of the C natural types for dates instead of a `long long`.

There are 2 approaches:
- the result is a Epoch timestamp: `time_t`
- the result is a Epoch-millis timestamp: `struct timeval`
- the result is a broken down date: ` struct tm`